tensorflow_graphics/projects/pointnet/train.py

```diff
@@ -135,8 +136,15 @@ def evaluate():
   num_examples = info.splits["train"].num_examples
   ds_train = ds_train.shuffle(num_examples, reshuffle_each_iteration=True)
   ds_train = ds_train.repeat(FLAGS.num_epochs)
-  ds_train = ds_train.batch(FLAGS.batch_size)
-  ds_test = ModelNet40.load(split="test").batch(FLAGS.batch_size)
+  ds_train = ds_train.map(functools.partial(preprocess, augment=FLAGS.augment),
+                          tf.data.experimental.AUTOTUNE)
+  ds_train = ds_train.batch(FLAGS.batch_size).prefetch(
+      tf.data.experimental.AUTOTUNE)
```

Comment:
#NoNewLines :)

Reply:
what's google's column limit? I'm using `yapf` with `{based_on_style: google, indent_width: 2}`

Reply:
Column limit is 80